src/axom/quest/Delaunay.hpp

```diff
@@ -395,20 +395,22 @@ class Delaunay
     return valid;
   }
 
-private:
   /// \brief Find the index of the element that contains the query point, or the element closest to the point.
```

Comment:
Note: The only real change to the Delaunay class to support this capability was in exposing the `findContainingElement()` and the `getBaryCoords()` functions as public instead of private.

Since we now expect some points to be outside of a valid cell, I added a second optional parameter to disable warnings in this case, but kept this warning on by default.